docs/deployments.md

```diff
@@ -0,0 +1,19 @@
+# Showing GitHub deployments in Jira
+
+The GitHub for Jira app automatically makes deployments created against [GitHub's deployment API](https://docs.github.com/en/rest/reference/repos#deployments) visible in Jira issues:
+
+![Deployments in Jira](./images/deployments-in-jira.png)
+
+To associate a deployment with a Jira issue, **the app looks for Jira issue keys in all commit messages on the deployed branch since the last successful deployment**.
+
+Let's look at an example:
+
+![Builds are associated by putting Jira issue keys into commit messages](./images/associating-deployments.png)
+
+We have two feature branches off the `main` branch. Say we have configured our GitHub Actions so that each push on the `main` branch triggers a deployment. 
```

Comment:
two feature branches _of_ the `main` branch

Reply:
each push _of_ the `main` branch

Reply:
no, I think "off the `main` branch" is correct. More correct would probably be "off of the main branch", but that sounds weird.

A push is going "onto" a branch, is it not?